meson.build

```diff
@@ -1,6 +1,6 @@
 project('gala',
     'c', 'vala',
-    version: '6.3.3',
+    version: '6.4.0',
```

Comment:
You have increased the version of Feedback to 7.0.0. Does that make sense here too?

Reply:
I'll spin up a VM today to test, but I think this might still be releasable on 6.1 maybe. If it contains breaking changes then yes we should bump to 7

Reply:
Okay yeah gonna bump to 7. It looks like we're relying on some newer icons and this doesn't actually fix some of the artifacting issues etc in OS 6, so probably not much point trying to backport it.